### PR TITLE
Create Git Repositories on Cloud article

### PR DESCRIPTION
**Pull request by eshanrnh**
I have created a draft article based on Tony's document (https://docs.google.com/document/d/1kjIaUz7MLXgle95X_8_ifKzxueP3xAvD9pR6LqnuLvY/edit).

### CODE REVIEW

**Comment by eshanrnh** [comment]
This PR needs to be further updated into 2 separate versions for 8 and below and 9 and above.

File: Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+versionFrom: 7.0.0
+versionTo: 10.0.0
+---
+
+# Repositories in an Umbraco cloud Project
+
+Each Umbraco cloud Project can have multiple environments: Development, Staging, and Live depending on your cloud project plan. Each environment has its own git repository that is hosted on Azure. Ideally, your Umbraco Cloud setup should look like this:
```
Comment (sophie-neale) [inline Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md:8]:
```suggestion
Each Umbraco cloud Project can have multiple environments: Development, Staging and Live depending on your cloud project plan. Each environment has it's own git repository that is hosted on Azure. Ideally, your Umbraco Cloud setup should look like this: 
```

File: Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [A source control repository with your own code](#a-source-control-repository-with-your-own-code)
+- [A Umbraco Cloud source control repository with the locally cloned Umbraco project](#a-git-umbraco-cloud-source-control-repository-with-the-locally-cloned-umbraco-project)
+
+:::note
+Umbraco cloud repositories are only deployment repositories and should not be used as source code repositories.
+:::
+
+## A source control repository with your own code
+
+Source control is a way to control changes to files and directories. You can keep a record of changes and revert to specific versions of a file in the event you would like to back up to an earlier time. A source control repository is used as the single source of truth that has the latest version of your project source code with all the git branches.
```
Comment (sophie-neale) [inline Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md:19]:
```suggestion
Source control is a way to control changes to files and directories. You can keep a record of changes and revert to specific versions of a file in the event you would like to back up to an earlier time. A source control repository is used as the single source of truth which has the latest version of your project source code with all the git branches.
```

File: Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+There are different source code management tools that you can use such as GitHub, Git, GitLab, Apache Subversion (SVN), CVS, Mercurial, etc.
+
+You need to put your custom code in a different source control repository you prefer. You can verify that it works on your local machine and copy those changes to the Umbraco Cloud source control repository with the locally cloned Umbraco project. The locally cloned Umbraco project has a `.gitignore` file. This file ignores all the Visual Studio temporary files, build results, files generated by popular Visual Studio add-ons, Umbraco folder, media files, log files, Packages and Plugins, etc. while pushing the changes to your cloud environments.
```
Comment (sophie-neale) [inline Umbraco-Cloud/Getting-Started/Repository-in-a-Cloud-Project/index.md:23]:
```suggestion
You need to put your custom code in a different source control repository of your choice. You can verify that it works on your local machine and copy those changes to the Umbraco Cloud source control repository with the locally cloned Umbraco project. The locally cloned Umbraco project has a `.gitignore` file. This file ignores all the Visual Studio temporary files, build results, files generated by popular Visual Studio add-ons, Umbraco folder, media files, log files, Packages and Plugins, etc. while pushing the changes to your cloud environments.
```

**Review by sophie-neale — COMMENTED** [review]
Hey @eshanrnh! Thanks so much for adding this! When I meet with the Docs Curator team on Monday we'll go through it 👍🏼 but for now, I've added some small suggestions 🙂

**Review by meyntony — APPROVED** [review]
Looks good :)

**Comment by eshanrnh** [comment]
Thanks for the quick review 🙌 Merging the PR in.

**Pull request merged**